pkg/exchange/okex/stream.go

```diff
@@ -42,9 +32,11 @@ type CandleKey struct {
 	Channel      string
 }
 
-func NewStream(client *okexapi.RestClient) *Stream {
+func NewStream(key, secret, passphrase string) *Stream {
```

Comment:
for constructors, we usually don't set optional arguments as required arguments.

optional arguments should be in a separated call

Reply:
ok, I added client back.